src/Doc/BuildDevDoc.cfg.in

```diff
@@ -279,7 +279,7 @@ TYPEDEF_HIDES_STRUCT   = NO
 # For small to medium size projects (<1000 input files) the default value is
 # probably good enough. For larger projects a too small cache size can cause
 # doxygen to be busy swapping symbols to and from disk most of the time
-# causing a significant performance penality.

```

Comment:
none of these doxy documentation errors exist upstream doxygen repo, so i fixed them here.